addons/hr_payroll/models/hr_payslip.py

```diff
@@ -271,24 +271,12 @@ def _sum_salary_rule_category(localdict, category, amount):
                     localdict = _sum_salary_rule_category(localdict, rule.category_id, tot_rule - previous_amount)
                     #create/overwrite the rule in the temporary results
                     result_dict[key] = {
+                        'sequence': rule.sequence,
+                        'code': rule.code,
+                        'name': rule.name,
+                        'note': rule.note,
                         'salary_rule_id': rule.id,
                         'contract_id': contract.id,
-                        'name': rule.name,
-                        'code': rule.code,
-                        'category_id': rule.category_id.id,
-                        'sequence': rule.sequence,
-                        'appears_on_payslip': rule.appears_on_payslip,
-                        'condition_select': rule.condition_select,
-                        'condition_python': rule.condition_python,
-                        'condition_range': rule.condition_range,
-                        'condition_range_min': rule.condition_range_min,
-                        'condition_range_max': rule.condition_range_max,
-                        'amount_select': rule.amount_select,
-                        'amount_fix': rule.amount_fix,
-                        'amount_python_compute': rule.amount_python_compute,
-                        'amount_percentage': rule.amount_percentage,
-                        'amount_percentage_base': rule.amount_percentage_base,
-                        'register_id': rule.register_id.id,
```

Comment:
Seems to be used here: report.hr_payroll.report_contributionregister

Should be stored as it is computed from a SQL Query.